coffea/nanoevents/factory.py

```diff
@@ -43,67 +44,6 @@ def __setstate__(self, state):
         self._cache = None
         self._events = lambda: None
 
-    @classmethod
```

Comment:
Can you update the tutorial notebook(s) to use `from_root`?

Reply:
done